inc/functions/i18n.php

```diff
@@ -323,7 +323,8 @@ function get_rocket_i18n_to_preserve( $current_lang ) { // phpcs:ignore WordPres
 	foreach ( $langs as $lang ) {
 		$parse_url           = get_rocket_parse_url( get_rocket_i18n_home_url( $lang ) );
 		$langs_to_preserve[] = _rocket_normalize_path(
-			"{$cache_path}{$parse_url['host']}(.*)/" . trim( $parse_url['path'], '/' )
+			"{$cache_path}{$parse_url['host']}(.*)/" . trim( $parse_url['path'], '/' ),
+			true // escape directory separators for regex.
```

Comment:
@Tabrisrp As this is a regex pattern, the directory separators need to be escaped. I made this change to ensure they are both normalized and escaped. Do you foresee any problems here? Why? The array it returns is different:

For Linux:

```
\/path\/to\/cache\/wp-rocket\/example.org(.*)\/fr
```

For IIS/Windows:

```
C:\\path\\to\\cache\\wp-rocket\\example.org(.*)\\fr
```

Reply:
As far as I can think of, it should be fine. Do you think using a different delimiter than `/` for the regex could help simplify this?

Reply:
>Do you think using a different delimiter than / for the regex could help simplify this?

I don't think it'll impact it and it's consistent with every where else in the code.